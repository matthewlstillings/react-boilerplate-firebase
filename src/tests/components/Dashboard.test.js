import React from 'react';
import {shallow} from 'enzyme';
import DashboardPage from '../../components/Dashboard';


test('should rener dashboard page', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
});