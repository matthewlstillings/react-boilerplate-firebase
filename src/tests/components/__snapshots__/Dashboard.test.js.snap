// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rener dashboard page 1`] = `
<div
  className="dashboard"
/>
`;
